### PR TITLE
[CHANGELOG] v4.0.0-alpha.30

### PR DESCRIPTION
**Pull request by dtassone**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,107 @@
...
+## [4.0.0-alpha.30](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.29...v4.0.0-alpha.30)
+
+_May 28, 2021_
+
+### @material-ui/x-grid@v4.0.0-alpha.30 / @material-ui/data-grid@v4.0.0-alpha.30
+
+Big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- ✅ Add `getCellClassName` prop (#1687) @m4theushw
+- 🇨🇿 Czech (csCZ) locale + Sloval plural rules fix (#1765) @Haaxor1689
```
Comment (m4theushw) [inline CHANGELOG.md:15]: We seem to list only the locale code in the highlights.

```suggestion
- 🇨🇿 Add csCZ locale (#1765) @Haaxor1689
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,107 @@
...
+- ⚡️ Remove `cellClassRules` from `GridColDef` (#1716) @m4theushw
+- 🐞 Fix date operators with date-time values (#1722) @m4theushw
+- 🐞 Fix pagination regressions issues (#1729) @ZeeshanTamboli
+- 🐞 Fix onFilterModelChange not firing (#1706) @dtassone
+- 🐞 More bugfixes
```
Comment (m4theushw) [inline CHANGELOG.md:20]:
```suggestion
- 🐛 Fix a regression in the controlled pagination (#1729) @ZeeshanTamboli
- 🐞 Bugfixes
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,107 @@
...
+- [docs] Fix 404 link (#1752) @oliviertassinari
+- [docs] Improve Custom edit component demo (#1750) @oliviertassinari
+- [docs] Remove redundant customizable pagination section (#1774) @ZeeshanTamboli
+- [docs] Update `GridApi` descriptions (#1767) @m4theushw
+
+### Core
+
+- [core] Batch updates of storybook (#1751) @oliviertassinari
+- [core] Help support different documents (#1754) @oliviertassinari
+- [core] upgrade material ui core v5 to latest version (#1763) @ZeeshanTamboli
```
Comment (m4theushw) [inline CHANGELOG.md:103]:
```suggestion
- [core] Upgrade Material-UI core v5 to latest version (#1763) @ZeeshanTamboli
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,107 @@
...
+
+- [DataGrid] Add `getCellClassName` prop (#1687) @m4theushw
+- [DataGrid] Add customisable `aria-label`, `aria-labelledby` field (#1764) @ZeeshanTamboli
+- [DataGrid] Czech (csCZ) locale + Sloval plural rules fix (#1765) @Haaxor1689
+- [DataGrid] Fix `onPageChange` doesn't update the `page` when a pagination button is clicked (#1719) @ZeeshanTamboli
+- [DataGrid] Fix cell accessibility aria-colindex (#1669) @ZeeshanTamboli
+- [DataGrid] Fix changing rows per page size (#1729) @ZeeshanTamboli
+- [DataGrid] Fix date operators not working with date-time values (#1722) @m4theushw
+- [DataGrid] Fix rowCount prop updates (#1697) @dtassone
+- [DataGrid] Improve deDe translation of \"errorOverlayDefaultLabel\" (#1718) @sebastianfrey
```
Comment (m4theushw) [inline CHANGELOG.md:85]:
```suggestion
- [DataGrid] Improve German (deDe) translation of "errorOverlayDefaultLabel" (#1718) @sebastianfrey
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,107 @@
...
+```diff
+-const { value } = apiRef.current.getEditCellValueParams(id, field);
++const { props: { value } } = apiRef.current.getEditCellPropsParams(id, field);
+```
+
+#### Changes
+
+- [DataGrid] Add `getCellClassName` prop (#1687) @m4theushw
+- [DataGrid] Add customisable `aria-label`, `aria-labelledby` field (#1764) @ZeeshanTamboli
+- [DataGrid] Czech (csCZ) locale + Sloval plural rules fix (#1765) @Haaxor1689
```
Comment (m4theushw) [inline CHANGELOG.md:79]:
```suggestion
- [DataGrid] Add Czech (csCZ) locale and fix plural rules in Slovak (skSK) (#1765) locale @Haaxor1689
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,107 @@
...
 
+## [4.0.0-alpha.30](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.29...v4.0.0-alpha.30)
+
+_May 28, 2021_
+
+### @material-ui/x-grid@v4.0.0-alpha.30 / @material-ui/data-grid@v4.0.0-alpha.30
+
+Big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- ✅ Add `getCellClassName` prop (#1687) @m4theushw
```
Comment (m4theushw) [inline CHANGELOG.md:14]:
```suggestion
- 💅 Add `getCellClassName` prop (#1687) @m4theushw
```

or 

```suggestion
- 💄 Add `getCellClassName` prop (#1687) @m4theushw
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,107 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.30](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.29...v4.0.0-alpha.30)
+
+_May 28, 2021_
+
+### @material-ui/x-grid@v4.0.0-alpha.30 / @material-ui/data-grid@v4.0.0-alpha.30
```
Comment (oliviertassinari) [inline CHANGELOG.md:10]:
```suggestion
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,107 @@
...
+Big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- ✅ Add `getCellClassName` prop (#1687) @m4theushw
+- 🇨🇿 Czech (csCZ) locale + Sloval plural rules fix (#1765) @Haaxor1689
+- ⚡️ Remove `cellClassRules` from `GridColDef` (#1716) @m4theushw
+- 🐞 Fix date operators with date-time values (#1722) @m4theushw
+- 🐞 Fix pagination regressions issues (#1729) @ZeeshanTamboli
+- 🐞 Fix onFilterModelChange not firing (#1706) @dtassone
+- 🐞 More bugfixes
+
```
Comment (oliviertassinari) [inline CHANGELOG.md:21]:
```suggestion
### @material-ui/x-grid@v4.0.0-alpha.30 / @material-ui/data-grid@v4.0.0-alpha.30
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -3,6 +3,107 @@
...
+```diff
+-<MuiDataGridFilterToolbarButton />
++<MuiDataGridToolbarFilterButton />
+-<GridColumnsToolbarButton />
++<GridToolbarColumnsButton />
+-<GridFilterToolbarButton />
++<GridToolbarFilterButton />
+-<GridDensitySelector />
++<GridToolbarDensitySelector />
+```
```
Comment (oliviertassinari) [inline CHANGELOG.md:35]:
````suggestion
```diff
-<MuiDataGridFilterToolbarButton />
+<MuiDataGridToolbarFilterButton />
```
```diff
-<GridColumnsToolbarButton />
+<GridToolbarColumnsButton />
```
```diff
-<GridFilterToolbarButton />
+<GridToolbarFilterButton />
```
```diff
-<GridDensitySelector />
+<GridToolbarDensitySelector />
```
````

**Review by oliviertassinari — COMMENTED** [review]
If we could keep the PR with the breaking changes together, it would be 👌

File: CHANGELOG.md [hunk 10]
```diff
@@ -3,6 +3,109 @@
...
+```diff
+-const { value } = apiRef.current.getEditCellValueParams(id, field);
++const { props: { value } } = apiRef.current.getEditCellPropsParams(id, field);
+```
+
+#### Changes
+
+- [DataGrid] Add `getCellClassName` prop (#1687) @m4theushw
+- [DataGrid] Add customisable `aria-label`, `aria-labelledby` field (#1764) @ZeeshanTamboli
+- [DataGrid] Add Czech (csCZ) locale and fix plural rules in Slovak (skSK) (#1765) locale @Haaxor1689
```
Comment (m4theushw) [inline CHANGELOG.md:84]: My previous suggestion was wrong.

```suggestion
- [DataGrid] Add Czech (csCZ) locale and fix plural rules in Slovak (skSK) locale (#1765) @Haaxor1689
```

**Review by m4theushw — APPROVED** [review]
Just tiny changes to keep consistency and it's good to go. :)

**Pull request merged**